modules/claims_api/app/swagger/claims_api/form_0966_v0_controller_swagger.rb

```diff
@@ -109,7 +109,7 @@ class Form0966V0ControllerSwagger
           key :name, 'X-VA-User'
           key :in, :header
           key :description, 'VA username of the person making the request'
-          key :required, true
+          key :required, false
```

Comment:
This seems like a significant change, but hard to review if its ok or not w/o subject area knowledge. Are there some good candidate reviewers?

Reply:
We've had a few refactors since this code was first introduced but the docs have failed to reflect this, thanks for pointing it out though.